project-templates/ecs/basic/README.md

```diff
@@ -0,0 +1,161 @@
+![img_10.png](../readme-images/tfc_remote_sharing_ss.png)
+
+
+3. Create a [global variable set](https://developer.hashicorp.com/terraform/tutorials/cloud-get-started/cloud-workspace-create#create-the-workspace) for your AWS Credentials.
+
+### AWS Steps with Terraform
+
+Create the baseline infrastructure with the following steps in your CLI.
+
+1. Navigate to the `terraform-aws-example-network` directory found in this example and run `terraform init` and the `terraform apply` command.
+
+2. Navigate to the `terraform-aws-example-microservice-infra` directory found in this example and run `terraform init` and the  `terraform apply` command.
+
+
+## Waypoint Template Steps 
+
+TODO(Teresa): Replace figma images with proper screenshots once UI is ready.
+
+
+1. Set up your TFC Credentials in Waypoint. You will need a TFC team access token found in Terraform Cloud and your TFC Organization Name.
+![img.png](../readme-images/waypoint_tfc_creds_setup_ss.png)
+
+
+2. Create a Waypoint Project Template by filling in the fields in the Template Creation page:
+![img_1.png](../readme-images/waypoint_create_template_ss.png)
+
+
+
+3. Return to your Projects List Page. In the top right corner, select `Create Project`. From the dropdown menu, select `Create Project with template`:
+![img_3.png](../readme-images/waypoint_projects_list_ss.png)
+
+   
+4. Select the template you would like to use for this project.
+![img_4.png](../readme-images/waypoint_select_template_ss.png)
+ 
+   
+
+5. Fill in the name of your project under `Project Name`. Then click `Create Project.
+![img_6.png](../readme-images/waypoint_create_project_ss.png)
+
+   
+6. Review the project details of your newly created project! You're Ready to Go!
+![img_7.png](../readme-images/waypoint_project_details_ss.png)
+
+
+## Waypoint Template Steps via CURL commands
+
+Pre-Requisites
+1. Your NamespaceID
+2. HCP Waypoint Authentication Token
+3. TFC Authentication Token
+
+
+
+1. Set up your TFC Credentials in Waypoint
+
+```shell
+curl --location --request PUT 'https:// api.hashicorp.cloud:443/waypoint/2022-02-03/namespace/{{NAMESPACE_ID}}/tfcconfig' \
+--header 'Authorization: {{YOUR_HCP_Waypoint_AUTH_TOKEN}}' \
+--header 'Content-Type: application/json' \
+--data '{"tfc_config":{"organization_name":"hcp_waypoint_integration", "token":"{{YOUR_TFC_AUTH_TOKEN}}"}, "namespace_id":"{{NAMESPACE_ID}}"}'
+
+
+```
+
+2. Create a Waypoint Project Template
+
+```shell
+curl --location 'https:// api.hashicorp.cloud:443/waypoint/2022-02-03/namespace/{{NAMESPACE_ID}}/projecttemplate' \
+--header 'Authorization: {{YOUR_HCP_Waypoint_AUTH_TOKEN}}' \
+--header 'Content-Type: application/json' \
+--data '{
+  "project_template": {
+    "name": "{{SOME_TEMPLATE_NAME}}",
+    "summary": "{{SOME_TEMPLATE_SUMMARY}}",
+    "readme_markdown_template": "{{SOME_TEMPLATE_README_TEXT}}",
+    "waypoint_project": {
+      "waypoint_hcl_template": "{{ENCODED_PROJECT_TEMPLATE_BYTES}}"
```

Comment:
Can you give an example of what to enter for `{{ENCODED_PROJECT_TEMPLATE_BYTES}}`? I'm confused how to run this one

Reply:
Just paired with @jgwhite, can be empty string to make the curl run, or something like `echo hello | base64` ... (`echo {{waypoint.hcl code}} | base64`)

Reply:
I will update this! Thank you!

Reply:
@teresamychu example is a-ok, I just wasn't sure what to put as the sample, if you want to keep `"{{ENCODED_PROJECT_TEMPLATE_BYTES}}"`, great with me. Going to suggest one more change now that the readme_markdown_template is bytes too

I need to set up Postman!

Reply:
Thank you Amy for coming back to this and updating the markdown template change! Really appreciate it!